projects/rero/ng-core/src/lib/record/search/record-search.component.ts

```diff
@@ -494,4 +485,13 @@ export class RecordSearchComponent implements OnInit {
       })
     );
   }
+
+  /**
+   * Return current URL after removing query parameters and updating resource type.
+   *
+   * @returns Updated url without query string
+   */
+  private getCurrentUrl(): string {
+    return this.router.url.split('?')[0].replace(/(^.*\/)(.*)$/, `$1${this.currentType}`);
```

Comment:
You can use angular library to extract the base URL.

Reply:
Done! But I'm not really sure that's the most efficient way.